packages/customer/src/repositories/address.ts

```diff
@@ -1,11 +0,0 @@
-import { DALUtils } from "@medusajs/utils"
```

Comment:
todo: In order to be future prood and make it easier, I suggest to create the repositories types in the module types as I did for all the other modules. there is multiple benefits to it
- if someone need to inject it in a service, he can use the repository type to type the repository and therefore align the approach accross the modules
- It also act as a self documentation
- If someone replace the DAL later, then he can implement the interface I am talking about

Reply:
I would argue that adding types that might become useful in the future for 1 out of 100 repositories is not worth it and will only hold us back from shipping modules fast.